ecs/volumes.go

```diff
@@ -42,7 +42,7 @@ func (b *ecsAPIService) createNFSmountIngress(securityGroups []string, project *
 				}
 				break
 			}
-			name := fmt.Sprintf("%sNFSMount%s", normalizeResourceName(s.Name), n)
+			name := fmt.Sprintf("%sNFSMount%s", normalizeResourceName(s.Name), normalizeResourceName(n))
```

Comment:
(not for this PR, but it looks like using ` normalizeResourceName()` can lead to ambiguity; https://github.com/docker/compose-cli/blob/ced0eb3912c8ff18d5245775be04098127e93ec5/ecs/cloudformation.go#L442-L444

For example, . `hello world`, `h e l l o world`, `h$%e^&lloworld`, `hello-world`, `Hello___world`, and `h
	e
	l
	l
	o
	
	
    world!!!!`

Are all considered equal. (see https://play.golang.org/p/Lp3xWkxztpy)

Is there a validation step before we reach this code (and other places where we normalise?); I'm always a bit scary with silently "fixing up" user-mistakes (if possible prefer to produce a clear, but "fatal" error to not hide mistakes).